tests/integration/components/validated-form-defaults-test.js

```diff
@@ -13,18 +13,16 @@ module("Integration | Component | validated form defaults", function (hooks) {
 
     this.set("model", { error: { test1: { validation: ["Error"] } } });
 
-    await render(hbs`
-      <ValidatedForm as |f|>
-        <f.input
-          @name="test1"
-          @type="text"
-          @label="Label!"
-          @hint="Hint!"
-          @submitted={{true}}
-          @model={{this.model}}
-        />
-      </ValidatedForm>
-    `);
+    await render(hbs`<ValidatedForm as |f|>
+  <f.input
+    @name="test1"
+    @type="text"
+    @label="Label!"
+    @hint="Hint!"
+    @submitted={{true}}
+    @model={{this.model}}
+  />
+</ValidatedForm>`);
```

Comment:
Indentation looks a little off :thinking:

Reply:
That's template lint

Reply:
Obviously, but why does it this strange formatting?

Reply:
I think it doesn't care about the JS indent because it's only concerned about the HBS inline style

Reply:
Looks ugly, but okey. :man_shrugging: